### PR TITLE
TSAPPS-179 Offers import to Tradeshift logic

### PR DESCRIPTION
**Pull request by tpischasova-str**
Offers from list should be "imported" to Tradeshift:
- Should be checked is buyer connected with supplier
- Should be checked - does offer exists. If not -it should be created with name from list
- if Offer exists, do nothing (later will be implemented logic of offer update)

### CODE REVIEW

File: externalAPI/tradeshiftAPI/supplierAPI.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+func (t *TradeshiftAPI) SearchOffer(name string) (map[string]interface{}, error) {
+	method := "/product-engine/supplier/supplier/v1/offers"
+	limit := 5
+
+	params := []rest.UrlParam{
+		{
+			Key:   "advancedSearch",
+			Value: fmt.Sprintf("{\"name\":\"%v\"}", name),
```
Comment (AndrewKolpakov) [inline externalAPI/tradeshiftAPI/supplierAPI.go:82]: name value should be quoted

File: main.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func main() {
 func start(
 	config *config.Config,
 	mapHandler mapping.HandlerInterface,
-	httpHandler http.Handler,
+	//httpHandler http.Handler,
```
Comment (AndrewKolpakov) [inline main.go:37]: It's good practice to remove unused code.

**Review by AndrewKolpakov — APPROVED** [review]
Just 2 minor comments and LGTM

File: main.go (hunk 2 above)
Comment (tpischasova-str) [reply main.go]: fixed in TSAPPS-176

File: externalAPI/tradeshiftAPI/supplierAPI.go (hunk 1 above)
Comment (tpischasova-str) [reply externalAPI/tradeshiftAPI/supplierAPI.go:82]: it is 
`\"%v\"}`

**Pull request merged**